server/routes/shared/showReferralPresenter.test.ts

```diff
@@ -142,10 +145,9 @@ describe(ShowReferralPresenter, () => {
   })
 
   describe('probationPractitionerTeamDetails', () => {
-    it('returns a summary list of probation practitioner team details', () => {
-      const sentReferral = sentReferralFactory.build(referralParams)
-      const presenter = new ShowReferralPresenter(
-        sentReferral,
+    function callShowReferralPresenter(staffDetails: DeliusStaffDetails) {
```

Comment:
I wonder if there's a better name for this function, to specify that the only difference is the staff details, something like `createShowReferralPresenterWithStaffDetails` might be clearer?

Reply:
Sure. I've used your suggestion.